### PR TITLE
Add Rules Unit Testing v2 skeleton for v9 release

### PR DESCRIPTION
**Pull request by yuchenshi**
Impl code is in other PRs and reviewed separately.

### CODE REVIEW

File: packages/rules-unit-testing/index.ts [hunk 1]
```diff
@@ -21,19 +21,4 @@
...
-  firestore,
-  initializeAdminApp,
-  initializeTestApp,
-  loadDatabaseRules,
-  loadFirestoreRules,
-  loadStorageRules,
-  useEmulators,
-  withFunctionTriggersDisabled
-} from './src/api';
+export const TODO = 'TODO';
```
Comment (Feiyang1) [inline packages/rules-unit-testing/index.ts:24]: Is it intentional?

File: packages/rules-unit-testing/package.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "@firebase/rules-unit-testing",
-  "version": "1.3.15",
+  "version": "2.0.0",
```
Comment (Feiyang1) [inline packages/rules-unit-testing/package.json:3]: No need to change the version. Add a changeset with major bump instead.

File: packages/rules-unit-testing/package.json [hunk 3]
```diff
@@ -19,25 +19,12 @@
...
-    "firebase-admin": "9.9.0",
-    "firebase-tools": "9.14.0",
-    "firebase-functions": "3.14.1",
     "rollup": "2.52.2",
     "rollup-plugin-typescript2": "0.30.0"
   },
   "peerDependencies": {
-    "@google-cloud/firestore": "^4.2.0",
-    "firebase-admin": "^9.7.0"
+    "firebase-exp": "^9.0.0-beta.8"
```
Comment (Feiyang1) [inline packages/rules-unit-testing/package.json:27]:
```suggestion
    "firebase": "^8.10.0"
```

changeset will update it to v9 automatically when publishing

**Review by Feiyang1 — CHANGES_REQUESTED** [review]

File: packages/rules-unit-testing/index.ts (hunk 1 above)
Comment (yuchenshi) [reply packages/rules-unit-testing/index.ts:24]: Yes, it needs to export something to make TypeScript happy. The real implementation is being rebased in a following PR

Comment (Feiyang1) [reply packages/rules-unit-testing/index.ts:24]: Do you intend to merge implementation into this PR?

Comment (yuchenshi) [reply packages/rules-unit-testing/index.ts:24]: Nope, see other PRs tagged with testing, which will be merged separately

**Review by Feiyang1 — APPROVED** [review]

**Pull request merged**